arangod/Aql/grammar.h

```diff
@@ -1,4 +1,4 @@
-/* A Bison parser, made by GNU Bison 3.7.6.  */
```

Comment:
hm, generated with an older bison?

Reply:
Strange that I have recently installed it in my computer, I'll try to update it and generate the files again and push here

Reply:
Recompiled it